### PR TITLE
PRL- 6384

### PR DESCRIPTION
**Pull request by sealDebolina**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/PRL-6384

### Change description ###

PRL-6384:AWP for citizen- C2- Requesting an order related to child- Fee correction

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/prl/services/FeeService.java [hunk 1]
```diff
@@ -174,13 +182,23 @@ && isFl403ApplicationAlreadyPresent(caseData)) {
         return feeType;
     }
 
+    private FeeType getFeeTypeByApplicationReason(FeeRequest feeRequest) {
+        return isc2WithOrder(feeRequest.getApplicationReason()) ? CHILD_ARRANGEMENTS_ORDER : C2_WITH_NOTICE;
+    }
+
+    public boolean isc2WithOrder(String applicationReason) {
+        return applicationReason.equals(CHILD_ARRANGEMENTS_ORDER_TO_LIVE_SPEND_TIME.getId())
```
Comment (channa-patil) [inline src/main/java/uk/gov/hmcts/reform/prl/services/FeeService.java:190]: Please put constants first then check for equals

Comment (sealDebolina) [reply src/main/java/uk/gov/hmcts/reform/prl/services/FeeService.java]: address RC

File: src/main/java/uk/gov/hmcts/reform/prl/enums/AwpApplicationReasonEnum.java [hunk 2]
```diff
@@ -43,8 +43,8 @@ public enum AwpApplicationReasonEnum {
     ),
     @JsonProperty("SPECIFIC_ISSUE_ORCDER")
     SPECIFIC_ISSUE_ORCDER(
-        "specfic-issue-order",
-        "specfic-issue-order"
+        "specific-issue-order",
```
[the remark this answers is not in the document]
Comment (sealDebolina) [reply src/main/java/uk/gov/hmcts/reform/prl/enums/AwpApplicationReasonEnum.java:46]: As we discussed, we do not need any changes in ccd.

[the remark this answers is not in the document]
Comment (channa-patil) [reply src/main/java/uk/gov/hmcts/reform/prl/enums/AwpApplicationReasonEnum.java:46]: These are the awp application reasons sent from citizen to COS for fee calculation, no CCD change is required.

[the remark this answers is not in the document]
Comment (sealDebolina) [reply src/main/java/uk/gov/hmcts/reform/prl/enums/AwpApplicationReasonEnum.java:46]: @channa-patil: checked ID value of enum with citizen frontend constants and no discrepency found.
Also fixed exsisting spelling mistakes in enums.

**Review by channa-patil — APPROVED** [review]

**Comment by sealDebolina** [comment]
feature/PRL-6384-AWP-C2-with-order branch have base changes for Awp c2 order related fee change(£184 to £255),
feature/PRL-6702 branch have fee following change, c2 order related fee will not change based on consent/notice.

In order to avoid re review please merge feature/PRL-6702 to feature/PRL-6384-AWP-C2-with-order, then feature/PRL-6384-AWP-C2-with-order to master.

Please merge feature/PRL-6702 in this PR before merging to master

@vivek-sekhar @shashisk97

**Review by shashisk97 — APPROVED** [review]

**Pull request merged**